tensorflow/compiler/mlir/quantization/tensorflow/python/BUILD

```diff
@@ -176,12 +176,8 @@ cc_library(
         "//tensorflow/compiler/mlir/quantization/tensorflow/calibrator:calibration_statistics_proto_cc",
         "//tensorflow/core/protobuf:for_core_protos_cc",
         "@com_google_absl//absl/container:flat_hash_map",
-<<<<<<< HEAD
         #"@com_google_absl//absl/strings:string_view",
         "@pybind11",
-=======
-        "@com_google_absl//absl/strings:string_view",
```

Comment:
Hmm. I think we can now pick the upstream one here.

Reply:
Fixed